templates/index.html

```diff
@@ -6,17 +6,17 @@
 
     <title>botamusique web interface</title>
 
-    <link rel="icon" href="static/image/favicon.ico" />
+    <link rel="icon" href="../static/image/favicon.ico" />
```

Comment:
Oh, are you sure this work?

Reply:
Lol, I've been just editing the static file directly. How does your normal development setup look like for this project?

Reply:
I just uncommemt L65 of interface.py then you can just refresh the and see your changes on the fly ;)

Reply:
Oh you mean you haven't get the bot running.... Then I have no good idea...

Reply:
> Oh you mean you haven't get the bot running.... Then I have no good idea...

Yeah, at least not on my (default) Windows rig. I run it on my CentOS box that hosts my Murmur server though.

I'll keep the template auto-refresh in mind :see_no_evil: